Makefile

```diff
@@ -278,16 +277,7 @@ endif
 
 .PHONY: gen/csv
 gen/csv:
```

Comment:
does it still make sense to call it `gen/csv`, as it looks like the script is doing more than just that?

Reply:
`gen/csv` was always doing more than just generating a csv, but i agree it is named badly. Maybe `release/prepare` to match the script, `release` or `release/create` might make more sense?

@StevenTobin wdyt?

Reply:
Yeah `release/prepare` makes sense, I think `release` or `release/create` implies it does more than just create and edit files locally.

I'll make that change

Reply:
@StevenTobin Yeah true, `release/prepare` seems fine to me. @wei-lee That work for you?

When you're updating can rebase master into your branch as well. Lot of flake e2e tests at the moment, so if by some miracle they pass i'd like it to merge asap.